tests/integ/test_search_expression.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from smexperiments.search_expression import Filter, Operator, SearchExpression, NestedFilter
+from smexperiments.experiment import Experiment
+import pytest
+
+
+def test_search(sagemaker_boto_client):
+    experiment_names_searched = []
+    search_filter = Filter(name="ExperimentName", operator=Operator.CONTAINS, value="smexperiments-integ-")
+    search_expression = SearchExpression(filters=[search_filter])
+    for s in Experiment.search(
+        search_expression=search_expression.to_boto(), max_results=10, sagemaker_boto_client=sagemaker_boto_client
```

Comment:
can we avoid calling code using "to_boto"?  Seems like the usage would be cleaner without it.

Reply:
agree and removed the need for to_boto here